charts/cmc-citizen-frontend/values.preview.template.yaml

```diff
@@ -95,7 +95,7 @@ cmc-claim-store:
   enabled: true
   java:
     imagePullPolicy: Always
-    image: hmctspublic.azurecr.io/cmc/claim-store:latest
+    image: hmctspublic.azurecr.io/cmc/claim-store:pr-3308
```

Comment:
Hi Tanya, Is claim store changed ? as per CIV-572, Claim store will also be effected. if so please provide me a PR number for claim store, otherwise , i will use latest.